### ISSUE TITLE
Shear scene

### ISSUE THREAD

**Issue by sidneymau** [des-science/y6-image-sims#2]
Configs that use random galaxy placements should have `shear_scene: True`—check this and validate the implementation.

**Comment by sidneymau** [comment]
Running `rand-bright.yaml` and `rand-bright-noshearscene.yaml` for tile `DES0402-2458` with seed `26598`

**Comment by sidneymau** [comment]
trying to understand if this impressionistic image is what we expect here... (note that color is asinh scaled)
![image](https://github.com/des-science/y6-image-sims/assets/26752606/d3888588-6ed1-4ffb-a54e-ca3d37cb04ea)

**Comment by sidneymau** [comment]
I don't understand why the images are so different between the configs, though
![image](https://github.com/des-science/y6-image-sims/assets/26752606/3ed35761-71c1-40c8-be05-3bef4906547a)
the only difference is `shear_scene: True` vs `shear_scene: False`

**Comment by sidneymau** [comment]
note: the darker spots are stars, and the fainter spots are galaxies (they are each being drawn at fixed magnitudes). so the stars shouldn't be moving around

**Comment by beckermr** [comment]
Yeah there's a bug alright.

**Comment by sidneymau** [comment]
Should [this code](https://github.com/esheldon/galsim_extra/blob/master/galsim_extra/mixed_scene.py#L70-L88) be checking for `obj_type`? I thought the idea was to apply shear to objects of type `gal` but not `star`

**Comment by beckermr** [comment]
The sheared images show way more sources. Let's figure out that before we worry about object types.

**Comment by sidneymau** [comment]
I did another test where I set `shear_scene` to `True` for galaxies but `False` for stars in the config file. A few notes from this test:
- that solution works fine for applying the scene shearing to only galaxies and not stars -- the stars are unaffected here
- I'm not really sure how the shear scene code is affecting the galaxies. I did some breakpoint checks and the number of objects, stamps, etc. didn't seem that different (beyond what you might expect from a few objects moving off the tile due to changed positions). I'll have to keep investigating

![image](https://github.com/des-science/y6-image-sims/assets/26752606/9985a3fd-5795-42d9-955d-e271c55009b3)
